client/src/components/Grid/GridElements/GridLink.vue

```diff
@@ -0,0 +1,10 @@
+<script setup lang="ts">
+interface Props {
+    text: string;
+}
+defineProps<Props>();
+</script>
+
+<template>
+    <a class="font-weight-bold" href="#" @click.prevent="$emit('click')">{{ text }}</a>
```

Comment:
```suggestion
    <button class="ui-link font-weight-bold" @click="$emit('click')">{{ text }}</button>
```

**Reason:**

Do not treat links like buttons for style purposes. Use the appropriate html element instead.